src/util/redis/redisConnector.js

```diff
@@ -34,6 +33,7 @@ const RedisDB = {
           log.error(`Redis is down at ${this.host}:${this.port}`);
           return false; // stop retrying
         },
+        tls: {}
```

Comment:
can this change go first & then enable the RBAC on redis cluster ?

Reply:
@anantjain45823

Reply:
Yes @krishna2020 this change will work even when RBAC is disabled or not there